ci-operator/jobs/infra-periodics.yaml

```diff
@@ -14,7 +22,7 @@ periodics:
       - --to-latest
       command:
       - /usr/bin/oc
-      image: registry.ci.openshift.org/ocp/4.12:cli
+      image: registry.ci.openshift.org/ocp/4.14:cli
```

Comment:
Why not `4.15`?
```
$ b01 version
Client Version: 4.13.0-202303281554.p0.g92b1a3d.assembly.stream-92b1a3d
Kustomize Version: v4.5.7
Server Version: 4.15.0-rc.5
Kubernetes Version: v1.28.6+0fb4726
```

Reply:
I get the impression that we always go with a latest GA version.